pkg/utils/kubernetes/health/extensions_test.go

```diff
@@ -115,6 +115,60 @@ var _ = Describe("Extensions", func() {
 				},
 				HaveOccurred(),
 			),
+			Entry("timestamp is before last update time",
```

Comment:
Can you add another test in which the truncated time is equal to the last update time?